sxw.c

```diff
@@ -392,11 +399,14 @@ void SXW_SW_Setup_Echo(void) {
 	CloseFile(&f);
 }
 
+/**
+ * \brief Get function for transpiration.
```

Comment:
change to: obtain transpiration (resource availability) for each resource group

Reply:
Addressed by commit 984bd13.